addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -221,6 +221,15 @@ var LivechatButton = Widget.extend({
                 this._renderMessages();
                 return;
             }
+            case 'mail.message/insert': {
+                const message = this._messages.find(message => message._id === payload.id);
+                if (!message) {
+                    return;
+                }
+                message._body = utils.Markup(payload.body);
```

Comment:
ci/security because of this. `payload.body` is body of `mail.message`, which is sanitised server-side so it's safe.